package.json

```diff
@@ -78,7 +78,7 @@
     "@artsy/icons": "3.22.1",
     "@artsy/img": "1.0.3",
     "@artsy/multienv": "^1.2.0",
-    "@artsy/palette": "^40.1.0",
```

Comment:
Manually bumped because of [this](https://artsy.slack.com/archives/C2TQ4PT8R/p1738253121649049) issue.